docs/getting_started.rst

```diff
@@ -15,9 +15,6 @@ If you encounter issues:
 * Searching online is often effective: chances are high that someone else encountered similar issues in the past
 * Please let us know if our docs can be improved, either by filing an issue or submitting a PR!
 
-.. note::
```

Comment:
Let's also keep this note, just in case, until we have further confirmation that the full Windows development workflow works without issues, thank you!